pkg/pillar/types/types.go

```diff
@@ -42,10 +42,11 @@ const (
 	PAUSED
 	HALTING // being halted
 	HALTED
-	RESTARTING // Restarting due to config change or zcli
-	PURGING    // Purging due to config change
-	BROKEN     // Domain is still alive, but its device model has failed
-	UNKNOWN    // State of the domain can't be determined
+	RESTARTING    // Restarting due to config change or zcli
+	PURGING       // Purging due to config change
+	BROKEN        // Domain is still alive, but its device model has failed
+	START_DELAYED // Domain start is delayed to due to StartDelay being honored.
```

Comment:
Sorry didn't see this earlier, but here it makes sense to keep things in logical order i.e., move the START_DELAYED just after AWAITNETWORKINSTANCE.

The protobuf values we can't change, but we have a function to do the mapping.